### PR TITLE
OSDOCS-715 pull secret doc improvements

### PR DESCRIPTION
**Pull request by bmcelvee**
https://jira.coreos.com/browse/OSDOCS-715

### CODE REVIEW

**Review by gabemontero — CHANGES_REQUESTED** [review]

File: modules/images-samples-credentials-options.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/configuring_samples_operator.adoc
+
+
+[id="images-samples-credentials-options_{context}"]
+= Configuring Samples Operator credentials
+
+You can create an additional secret in the {product-title} namespace, which will get used automatically for the Samples Operator.
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/images-samples-credentials-options.adoc:9]: Ah crumbs, that's right. Thank you!

File: modules/images-imagestream-import-images-private-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// * assembly/openshift_images/managing-image-streams.adoc
+
+[id="images-imagestream-import-images-private-registry_{context}"]
+= Importing images from private registries
+
+An imagestream can be configured to import tag and image metadata from private image registries, requiring authentication.
+
+[NOTE]
+====
+When importing from an insecure registry, the registry URL defined in the Secret must include the :80 port suffix or the Secret is not used when attempting to import from the registry.
```
Comment (xiuwang) [inline modules/images-imagestream-import-images-private-registry.adoc:11]: It's not only for insecure registry. Both insecure and secure registry need add port if they defined one.
such as “upshift.mirror-registry.qe.devcluster.openshift.com:5000” and “upshift.mirror-registry.qe.devcluster.openshift.com:5001”， they are secure registries but could be configured with different authication.
@gabemontero  Could you help confirm this NOTE ?

File: modules/images-imagestream-import-images-private-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+====
+
+.Procedure
+
+. You must create a Secret object that is used to store your credentials by entering the following command:
++
+----
+$ oc create secret generic <secret_name> \
+    --from-file=<file>.txt
```
Comment (xiuwang) [inline modules/images-imagestream-import-images-private-registry.adoc:20]: The cmd should be 
`oc create secret generic <secret_name> --from-file=.dockercfg=<file_absolute_path> --type=kubernetes.io/dockercfg`
or `oc create secret generic <secret_name> --from-file=.dockerconfigjson=<file_absolute_path> --type=kubernetes.io/dockerconfigjson`

e.g

`oc create secret generic test --from-file=.dockerconfigjson=/home/wxj/.docker/config.json --type=kubernetes.io/dockerconfigjson
secret/test created`

File: modules/images-imagestream-import-images-private-registry.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+secret/<secret_name>
+----
++
+. After the Secret is configured, create the new imagestream or enter the `oc import-image` command:
++
+----
+$ oc import-image <image-name>
```
Comment (xiuwang) [inline modules/images-imagestream-import-images-private-registry.adoc:28]: When create a new imagestream, we could use cmds:
`oc tag <image> <imagestreamtag>`
or 
`oc import-image <imagestreamtag> --from=<image> --confirm`

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 5]
```diff
@@ -14,6 +14,27 @@ However, for other scenarios, such as referencing images across {product-title}
 projects or from secured registries, then additional configuration steps are
 required.
 
+You can obtain the image pull secret, `pullSecret`, from the link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site.
+
+You use this pull secret to authenticate with the services that are provided by the included authorities, including Quay.io and registry.redhat.io, which serves the container images for {product-title} components.
```
Comment (xiuwang) [inline openshift_images/managing_images/using-image-pull-secrets.adoc:19]: s/Quay.io/quay.io

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 6]
```diff
@@ -14,6 +14,27 @@ However, for other scenarios, such as referencing images across {product-title}
...
+
+.Example config.json fole
```
Comment (xiuwang) [inline openshift_images/managing_images/using-image-pull-secrets.adoc:21]: s/fole/file

**Comment by xiuwang** [comment]
LGTM, thanks

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :fallen_leaf:

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 7]
```diff
@@ -6,14 +6,35 @@ include::modules/common-attributes.adoc[]
...
 already have the correct permissions and no additional action should be
 required.
 
 However, for other scenarios, such as referencing images across {product-title}
 projects or from secured registries, then additional configuration steps are
 required.
 
+You can obtain the image pull secret, `pullSecret`, from the link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site.
+
+You use this pull secret to authenticate with the services that are provided by the included authorities, including link:quay.io[Quay.io] and link:registry.redhat.io[registry.redhat.io], which serves the container images for {product-title} components.
```
Comment (jeana-redhat) [inline openshift_images/managing_images/using-image-pull-secrets.adoc:19]: s/serves/serve (unless I'm misunderstanding and that's not really plural :slightly_smiling_face:)

**Review by jeana-redhat — APPROVED** [review]
Small suggestions! Looks nice overall.

File: modules/images-imagestream-import-images-private-registry.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+[id="images-imagestream-import-images-private-registry_{context}"]
+= Importing images and imagestreams from private registries
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/images-imagestream-import-images-private-registry.adoc:5]: Crumbs. Forgot about that one. -_- It's flip-floped so many times. TY

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 9]
```diff
@@ -6,14 +6,35 @@ include::modules/common-attributes.adoc[]
...
+
+.Example config.json file
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply openshift_images/managing_images/using-image-pull-secrets.adoc:21]: Couldn't hurt :)

**Pull request merged**